taskcluster/android_taskgraph/transforms/push_apk.py

```diff
@@ -34,3 +34,13 @@ def resolve_keys(config, tasks):
                 }
             )
         yield task
+
+
+@transforms.add
+def add_startup_test(config, tasks):
+    for task in tasks:
+        if "nightly" not in task["attributes"].get("build-type", ""):
+            yield task
+            continue
+        task["dependencies"]["startup-test"] = "startup-test-nightly-arm"
+        yield task
```

Comment:
I'm probably going to rename [this](https://github.com/mozilla-mobile/firefox-android/blob/05e561dd8bbece23cf9914969f2efbb8c2ff6599/taskcluster/ci/startup-test/kind.yml#L49) to `fenix-arm-nightly` and add a second task underneath `focus-arm-nightly` and clean up the defaults etc

Reply:
Sounds good, I'm happy to change this to match the new tasks, and/or review your changes :)